### DISCUSSION TITLE
base packages for mingw-w64 repos?

### DISCUSSION THREAD

**Discussion by goyalyashpal** [msys2/MSYS2-packages#3595] [Q&A]
- While running install commands `-S` for even seemingly unrelated packages from `mingw-w64-*` repos (`vlc`, `clangd`, `kate` etc) 
- ... i noticed that there are several common packages 
- which are there in the `msys` install by default (`pactree -c base`) as well like `brotli`, `libcurl` etc...

So, is there some "base" package for those `mingw-w64` repos as well?

**Comment by Biswa96** [comment]
> So, is there some "base" package for those mingw-w64 repos as well?

No.

**Reply by goyalyashpal** [reply]
do u think t'd make sense?

**Reply by Biswa96** [reply]
OK, here is the long version. There is no "base" package for those mingw-w64 repos.

**Reply by goyalyashpal** [reply]
here's a long version:

do u think it will make sense to have base package in those repo to mimick base pkg of msys repo that is, a base pkg to contain those frequently used packages?

**Reply by Biswa96** [reply]
The base package group contains only the packages which are needed to run the pacman. And no mingw packages are used or required at the first install time. Users have to choose which packages to install. If you want to install multiple packages at once, there are some group packages listed here https://packages.msys2.org/groups/

**Reply by goyalyashpal** [reply]
i search groups already, and i didnt find such one...

> And no mingw packages are used or required at the first install time.

yes, my intention is not that they be installed bundled in, 

my intention for now is ease of comparison about which repo to choose. this is not the only one... though, i have to think more about what other benefit cases can be.

**Comment by MehdiChinoune** [comment]
Sorry to be honest!
Shouldn't this user be blocked, I see him asking many annoying questions and complaining about many things.

**Reply by goyalyashpal** [reply]
> I see him asking many annoying questions and complaining about many things.

i am never complaining, i genuinely would like u to cite me where i "complained about many things" in this repo - and be specific about what u say as complaining.

other than that, yes my questions might be annoying. but thats why i dont put those on issues - as only important actionable things have to go to issues.

but discussions - no, the devs are free to ignore it. i never pointed or mentioned anyone here in the discussions for their responses.

what are discussions for?
